addons/explosives/ExplosivesUI.hpp

```diff
@@ -80,7 +80,7 @@ class RscACE_SelectTimeUI {
             y = 0.06;
             w = 0.49;
             h = 0.025;
-            onSliderPosChanged = "private ['_mins', '_secs'];_mins = floor((_this select 1)/60);_secs=floor((_this select 1) - (_mins*60));ctrlSetText [8870, format[localize 'STR_ACE_Explosives_TimerMenu',_mins, _secs]];";
+            onSliderPosChanged = "_mins = floor((_this select 1)/60);_secs=floor((_this select 1) - (_mins*60));ctrlSetText [8870, format[localize 'STR_ACE_Explosives_TimerMenu',_mins, _secs]];";
```

Comment:
I guess these aren't private, but I don't think it matters for BIS event handlers as there shouldn't be any upper scope variables (I hope)

Reply:
They should still be privatized.

Reply:
Exactly why I removed that. There is no upper scope. So private does nothing in this case. Besides making the string longer. Thus increasing compileTime. And as this is a string eventhandler that get's compiled before every usage...